pkg/autoscaler/metrics/sample_size.go

```diff
@@ -43,8 +43,7 @@ const (
 //   n = N*X / (N + X – 1), X = C^2 * σ^2 / MOE^2,
 //
 // where N is the population size, C is the critical value of the Normal distribution
-// for a given confidence level of 95%, MOE is the margin of error and σ^2 is the
-// population variance.
+// for a given confidence level of 95%, MOE is the margin of error.
```

Comment:
wondering if it's intentional to lose the definition of `σ^2` here since it's still used in the pseudo-equation above?

Reply:
Good point!